src/docfx/build/legacy/LegacyUtility.cs

```diff
@@ -13,7 +13,7 @@ public static string ToLegacyOutputPathRelativeToSiteBasePath(this Document doc,
             var outputPath = manifestItem.Path;
             if (doc.ContentType == ContentType.Resource && !doc.Docset.Config.Output.CopyResources)
             {
-                outputPath = doc.GetOutputPath(manifestItem.Monikers, isPage: false);
+                outputPath = UrlUtility.Combine(docset.SiteBasePath, MonikerUtility.GetGroup(manifestItem.Monikers) ?? "", doc.SitePath);
```

Comment:
not reuse `documentProvider.GetOutputPath` here, do not want to involve `context` as dependency?

Reply:
yep